frontend/src/components/EasyModal.tsx

```diff
@@ -0,0 +1,60 @@
+"use client";
+
+import { RefObject } from "react";
+import RightCloseButton from "./RightCloseButton";
+import BaseModal from "./BaseModal";
+import PrimaryButton from "./PrimaryButton";
+import { Check } from "react-feather";
```

Comment:
Usikker på om Check skal være med i denne komponenten, for ser den ikke på designet

Reply:
Er det greit at den lille importen ligger igjen der?

Har jobbet meg ut ifra denne branchen nå hvor EasyModal blir ryddet enda mer opp i, og ser det blir litt kluss og reverte arbeidet mitt/stashe m.m. for å løse det.

Reply:
Det går fint ja 👍